lib/car.rb

```diff
@@ -0,0 +1,26 @@
+class Car
+
+  attr_accessor :make, :model, :monthly_payment, :loan_length, :color
+
+  def initialize(make_and_model, monthly_payment, loan_length)
+    @monthly_payment = monthly_payment
+    @loan_length = loan_length
+    make_and_model = {
+      make: @make,
+      model: @model
+      }
+    end
+
+
+  def color
```

Comment:
Pay close attention to the interaction pattern. The return value for `color` was `nil` not and empty array.
```
pry(main)> car.color
#=> nil
```